src/main/webapp/app/exercises/shared/feedback/feedback.component.ts

```diff
@@ -180,6 +180,8 @@ export class FeedbackComponent implements OnInit {
                 switchMap((feedbacks: Feedback[] | undefined | null) => {
                     if (feedbacks && feedbacks.length) {
                         this.result.feedbacks = feedbacks!;
+                        feedbacks.forEach((feedback) => (feedback.result = this.result));
```

Comment:
Why is this necessary? The feedback results were set in `resultService.getFeedbackDetailsForResult`, `feedbackService.getFeedbacksForResult` (change above) and now here

Reply:
You are right, it is only needed if the feedbacks are retrieved via the `result` input to the component. In that case they might not be connected properly. I moved this line up into the relevant case.